alignak/shinken_import_hook.py

```diff
@@ -1,5 +1,5 @@
-
 # -*- coding: utf-8 -*-
```

Comment:
simply because it's not a *script*, it's a simple python *module*, as are 99(.9)% of the *.py* files  in the project.  And so should not be directly **executed** as a python script.
Even the daemons *main* modules (which are actually under `bin` : `arbiter-daemon.py` and the others..) (but that will be (when the setup PR will be done)) in *alignak/bin/* (or more precisely in `alignak.bin`) won't require the shebang, as they will be **imported** (not directly *run/executed*) by the console_entry scripts setup by the .. setup(pbr) ;)

*basically* : if there is no `if __name__ == __main__:` in a module then it is not a *script* (or should not be treated as one). And even when there such a construction it's not always necessary/required/desired to have the shebang, like when you want to execute a package (`python -m package(.sub..)`).